source/core/ControlRef.cpp

```diff
@@ -54,34 +54,42 @@ NIError ControlReferenceDestroy(ControlRefNum refnum) {
 // Clean-up Proc, run when top level VI finishes, disposing control reference.
 static void CleanUpControlReference(intptr_t arg) {
     ControlRefNum refnum = ControlRefNum(arg);
+    ControlRefInfo controlRefInfo;
```

Comment:
Please add this code to ControlReferenceDestroy and update the function comment.